### DISCUSSION TITLE
Release note of the start screen

### DISCUSSION THREAD

**Discussion by resigned1292** [syl20bnr/spacemacs#15287] [Q&A] [answered]
![2022-01-25_19-51](https://user-images.githubusercontent.com/69884931/150972478-99b0fccd-0eaa-436f-bfda-c28727e7ae6d.png)

The release note shows that the version of my spacemacs is 0.999.x, and it is current rolling release.
Which update is this?? I mean I'm a little out of the habit...

**Comment by lebensterben** [comment, marked as answer]
We've recently changed the version number to reflect that the develop branch is constantly being updated  for a few years and should be treated as a default branch.

**Reply by levone1** [reply]
Sorry if obvious - just want to make sure I'm not missing it...  So you're saying that the 0.999.0 version is that reflection, right?  Because it shows the same when using develop branch, (for me anyway).

Thanks
